xtask/src/main.rs

```diff
@@ -0,0 +1,277 @@
+use std::{env, error::Error};
+
+type DynError = Box<dyn Error>;
```

Comment:
We might as well use anyhow here, it provides better ergonomic.

Reply:
I wanted to keep this crate as small and dependency free as possible since it's an ad hoc task runner, I think we can move to anyhow if error handling becomes too unwieldy or important.